src/backend/base/langflow/services/database/service.py

```diff
@@ -117,9 +118,7 @@ def _get_connect_args(self):
         return connect_args
 
     def on_connection(self, dbapi_connection, _connection_record) -> None:
-        if isinstance(
-            dbapi_connection, sqlite3.Connection | sa.dialects.sqlite.aiosqlite.AsyncAdapt_aiosqlite_connection
-        ):
+        if isinstance(dbapi_connection, sqlite3.Connection | dialect_sqlite.aiosqlite.AsyncAdapt_aiosqlite_connection):
```

Comment:
I don't understand the difference. 

`sa.dialects.sqlite` is the same as `from sqlalchemy.dialects import sqlite`

Reply:
yes

Reply:
Weird. I tested with both SQlite and Postgres when introducing this change. And I also don't see the difference between the imports 🤔